.gitignore

```diff
@@ -1,3 +1,4 @@
 target/
 WEB-INF/appengine-generated/datastore-indexes-auto.xml
 .idea
+sendgrid.env
```

Comment:
This has my API key, but Git can't see it, that's why there's an error in the Maven test.
Need to use Secrets or smthg.